RecoTracker/TkSeedGenerator/plugins/JetCoreMCtruthSeedGenerator.h

```diff
@@ -0,0 +1,195 @@
+#ifndef RecoTracker_TkSeedGenerator_JetCoreMCtruthSeedGenerator_H
+#define RecoTracker_TkSeedGenerator_JetCoreMCtruthSeedGenerator_H
+
+#define jetDimX 30
```

Comment:
please add a comment to describe these numbers

Reply:
done in the last commit